components/layout_2020/flexbox/layout.rs

```diff
@@ -1033,7 +1069,7 @@ impl<'items> FlexLine<'items> {
     fn resolve_auto_main_margins(
         &self,
         remaining_free_space: Length,
-    ) -> impl Iterator<Item = (Length, Length)> + '_ {
+    ) -> (impl Iterator<Item = (Length, Length)> + '_, bool) {
```

Comment:
Maybe you could use

```rs
    fn resolve_auto_main_margins(
        &self,
        remaining_free_space: &mut Length,
    ) -> impl Iterator<Item = (Length, Length)> + '_ {
```

and just set `*remaining_free_space = Length::zero();` is there are auto margins.

Reply:
I'm not against this, but I wonder if this does not conform to the style of current code, which is quite functional.

Reply:
It's fine as-is, but then I guess you should update the comment above the function to mention the bool.

Reply:
Okay it is done.